--- denemo-2.4.0/src/core/utils.c.orig	2020-03-20 17:49:09.284852354 +0100
+++ denemo-2.4.0/src/core/utils.c	2020-07-03 18:44:47.365762199 +0200
@@ -1629,7 +1629,7 @@
 #else
       gchar *prefix = g_build_filename (get_prefix_dir (), NULL);
 #endif
-      fontdir = g_build_filename (prefix, "share", "fonts", "truetype", "denemo", NULL);
+      fontdir = g_build_filename (prefix, "share", "fonts", "denemo", NULL);
     }
   return fontdir;
 }
@@ -3523,7 +3523,7 @@
   dirs = g_list_append (dirs, g_build_filename (get_executable_dir (), "..", get_local_dir (dir), NULL));
   dirs = g_list_append (dirs, g_build_filename (get_user_data_dir (TRUE), get_local_dir (dir), NULL));
   dirs = g_list_append (dirs, g_strdup (get_system_dir (dir)));
-  dirs = g_list_append (dirs, g_build_filename (get_executable_dir (), "..", "share", "fonts", "truetype", "denemo", NULL));
+  dirs = g_list_append (dirs, g_build_filename (get_executable_dir (), "..", "share", "fonts", "denemo", NULL));
   return find_path_for_file (filename, dirs);
 }
 
--- denemo-2.4.0/fonts/Makefile.am.orig	2020-07-03 18:45:00.825689280 +0200
+++ denemo-2.4.0/fonts/Makefile.am	2020-07-03 18:45:03.192343125 +0200
@@ -1,7 +1,7 @@
 #dist_pkgdata_DATA = Denemo.ttf feta.ttf emmentaler.ttf
 
 
-fontsdir = $(datadir)/fonts/truetype/denemo
+fontsdir = $(datadir)/fonts/denemo
 
 fonts_DATA = $(srcdir)/Denemo.ttf  $(srcdir)/feta.ttf $(srcdir)/emmentaler.ttf
 
